buildSrc/subprojects/ide/src/main/kotlin/org/gradle/gradlebuild/ide/IdePlugin.kt

```diff
@@ -155,23 +155,23 @@ open class IdePlugin : Plugin<Project> {
     private
     fun ProjectSettings.configureRunConfigurations(rootProject: Project) {
         runConfigurations {
-            val gradleRunners = mapOf(
-                "Regenerate Int Test Image" to "prepareVersionsInfo intTestImage publishLocalArchives"
-            )
-            gradleRunners.forEach { (name, tasks) ->
-                create<Application>(name) {
-                    mainClass = "org.gradle.testing.internal.util.GradlewRunner"
-                    programParameters = tasks
-                    workingDirectory = rootProject.projectDir.absolutePath
-                    moduleName = "org.gradle.internalTesting.main"
-                    envs = mapOf("TERM" to "xterm")
-                    beforeRun {
-                        create<Make>("make") {
-                            enabled = false
-                        }
-                    }
-                }
-            }
+//            val gradleRunners = mapOf(
```

Comment:
I think we should remove the code because we can find them back from VCS at any time, right?

Reply:
Yes, I'll remove the code

Reply:
Removed.